src/Symfony/Component/Validator/Constraints/EmailValidator.php

```diff
@@ -118,7 +118,7 @@ public function validate($value, Constraint $constraint)
      */
     private function checkMX($host)
     {
-        return checkdnsrr($host, 'MX');
+        return !empty($host) && checkdnsrr($host, 'MX');
```

Comment:
Let's cast the substring result above (or perhaps explicitly check for false); as currently we literally get `string('false')` as a replacement for `{{ value }}` in the violations above; due https://github.com/apetitpa/symfony/blob/master/src/Symfony/Component/Validator/ConstraintValidator.php#L139

So we got an extra (tiny) bug here ;-)